fighters/packun/src/acmd/other.rs

```diff
@@ -128,21 +128,87 @@ unsafe fn appeal_hi_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     let cur_stance = VarModule::get_int(boma.object(), vars::packun::instance::CURRENT_STANCE);
-    VarModule::set_int(boma.object(), vars::packun::instance::CURRENT_STANCE, (cur_stance + 1) % 3);
-    VarModule::on_flag(fighter.object(), vars::packun::instance::STANCE_INIT);
-    frame(lua_state, 10.0);
+    let mut advance = 0;
+    let mut holding = true;
```

Comment:
I believe you'll need to use VarModule to properly track the `holding` flag e.g. `vars::packun::status::is_button_held`